be/src/exprs/runtime_filter.cpp

```diff
@@ -452,14 +457,29 @@ class RuntimePredicateWrapper {
                 << "Can not change to bloom filter because of runtime filter type is "
                 << to_string(_filter_type);
         _is_bloomfilter = true;
+        insert_to_bloom_filter(_bloomfilter_func.get());
+        // release in filter
+        _hybrid_set.reset(create_set(_column_return_type));
+    }
+
+    void insert_to_bloom_filter(BloomFilterFuncBase* bloom_filter) const {
         if (_hybrid_set->size() > 0) {
+            bool use_batch = _state->enable_vectorized_exec() &&
```

Comment:
use_batch seems can be set in constructor. maybe better `_use_batch` or tmeplate